Header header
string uuid
string world_trace
string world_qsr_trace
